GCTC-NTGC/gc-digital-talent
# Add option to remove orphaned i18n strings in check-intl-* scripts

Resolves #4305.

## Steps to test
1. Delete all **newTranslations.json** and **untranslated.json** files from project (these are untracked files and could be have leftover work from other branches)
2. Extract admin `npm run check-intl-admin --workspace=common`
3. Extract common `npm run check-intl-common --workspace=common`
4. Extract indigenousapprenticeship `npm run check-intl-indigenousapprenticeship --workspace=common`
5. Extract talentsearch `npm run check-intl-talentsearch --workspace=common`
6. Merge admin `npm run check-intl-admin-merge --workspace=common`
7. Merge common `npm run check-intl-common-merge --workspace=common`
8. Merge indigenousapprenticeship `npm run check-intl-indigenousapprenticeship-merge --workspace=common`
9. Merge talentsearch `npm run check-intl-talentsearch-merge --workspace=common`
10. Confirm strings removed in fr.json files are not present in codebase by searching all files for their IDs
